onboarding_project/squad_pantry_app/views.py

```diff
@@ -13,3 +15,25 @@ class OrderViewSet(viewsets.ModelViewSet):
 
     def get_queryset(self):
         return Order.objects.filter(placed_by=self.request.user)
+
+
+class MetricView(View):
+    template_name = 'admin/metrics.html'
+
+    def get(self, request):
+        if len(request.GET) > 0:
+            if request.GET['end_date'] <= request.GET['start_date']:
+                error = 'End Date should be more recent than Start Date'
+                return render(request, self.template_name, {'error': error})
+            throughput, turnaround_time = PerformanceMetrics.get_metrics_data(request.GET['start_date'],
+                                                                              request.GET['end_date'])
+            print(throughput, turnaround_time)
```

Comment:
No `print`s. _Should have  been caught in self review_

Reply:
Done!